tests/nodeos_snapshot_programmable_test.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+
```

Comment:
This is almost identical to `nodeos_snapshot_diff_test.py`. Seems like they could easily be combined into one test that does both.

Reply:
Thank you! I updated nodeos_snapshot_diff_test to use scheduler and then compare that snapshot to irreversible.  I am working on re-purposing nodeos_snapshot_programmable_test for fork scenario

Reply:
Are you planning on removing this test then?